src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -5509,7 +5509,7 @@ int RGWRados::delete_bucket(RGWBucketInfo& bucket_info, RGWObjVersionTracker& ob
     maybe_warn_about_blocking(dpp); // TODO: use AioTrottle
     (void) CLSRGWIssueBucketIndexClean(index_pool,
 				       bucket_objs,
-				       cct->_conf->rgw_bucket_index_max_aio)();
+				       cct->_conf->rgw_clean_bucket_index_max_aio)();
```

Comment:
Apart of better index cleanup configurability this effectively changes the default value (from 128 to 1). Is this really intended?